EasyPost.Tests/UnitTest.cs

```diff
@@ -0,0 +1,85 @@
+using System;
+using System.Threading.Tasks;
+
+namespace EasyPost.Tests
+{
+    /// <summary>
+    ///     Base class for all unit tests.
+    ///     Sets up all available client versions for VCR and non-VCR requests.
+    /// </summary>
+    public class UnitTest : IDisposable
+    {
+        private readonly TestUtils.VCR? _vcr;
+
+        protected Client Client;
+
+        private string? _cleanupId;
+
+        /// <summary>
+        ///     An asynchronous function executed after every test.
+        ///     Function should have a string (ID) passed in, and should return a boolean.
+        /// </summary>
+        protected Func<string, Task<bool>>? CleanupFunction { get; set; }
+
+#pragma warning disable CS8618
+        protected UnitTest(string groupName, TestUtils.ApiKey apiKey = TestUtils.ApiKey.Test) => _vcr = new TestUtils.VCR(groupName, apiKey);
+#pragma warning restore CS8618
+
+        public void Dispose()
```

Comment:
Issue: `Dispose` what? Function names should be `VerbNoun` for clarity.

This should probably also have a docstring attached.

Reply:
This is an inherited method from the interface, so we can't rename it. I will add a docstring.